zqd/api/api.go

```diff
@@ -112,9 +112,15 @@ type PacketPostStatus struct {
 
 type LogPostRequest struct {
 	Paths          []string             `json:"paths"`
+	StopErr        bool                 `json:"stop_err"`
 	JSONTypeConfig *ndjsonio.TypeConfig `json:"json_type_config"`
 }
 
+type LogPostWarnings struct {
+	Type     string   `json:"type"`
+	Warnings []string `json:"warnings"`
```

Comment:
Any reason not to simplify this to carry just a single warning?
```suggestion
type LogPostWarning struct {
	Type    string   `json:"type"`
	Warning string `json:"warning"`
```

Reply:
@nwt - see my comment above; this is to make it consistent with the existing search warnings response. Separate from the PR, we can modify the Brim client to just expect a single string.

Reply:
@nwt i went with @alfred-landrum's to be consistent with search warnings. https://github.com/brimsec/zq/pull/582#discussion_r408943752

we are both still uncertain about the need for a slice so maybe we should change this and search warning

Reply:
Let's move to a single string later, then.